client/src/components/Authorization/Login.js

```diff
@@ -192,20 +193,20 @@ const Login = props => {
                     </Link>
                   </div>
 
-                  <button
+                  <Button
                     id="cy-login-submit"
                     type="submit"
                     className="btn-submit"
```

Comment:
We should be getting rid of classname and using the default style of the Button component